### DISCUSSION TITLE
Transferring contributions from a work account to a private account

### DISCUSSION THREAD

**Discussion by mishal0488** [community/community#110601] [Other Feature Feedback, Questions, & Ideas]
Hi Guys

How does one transfer their contributions from one GitHub account to another?
I basically want my contributions from my work account to reflect on my personal GitHub account.

I have seen several posts on this with no resolution.

I have tried adding an email address however it just results in an error.
Can anyone please shed some light on the topic?

**Comment by git-sumana** [comment]
Response #1
Transferring contributions from one GitHub account to another isn't directly supported by GitHub due to privacy and security concerns.

**Comment by git-sumana** [comment]
Response #2
Better you fork and make PRs so that it would even help in rising the git-stats and git-grade

**Comment by git-sumana** [comment]
Response #3
 Another option is to include a link to your work GitHub account in your personal GitHub profile. This way, visitors can see your contributions on your work account even if they are not directly attributed to your personal account.
